pkg/apis/pulumi/v1alpha1/stack_types.go

```diff
@@ -57,7 +57,7 @@ type StackSpec struct {
 	SecretRefs map[string]ResourceRef `json:"secretsRef,omitempty"`
 	// (optional) SecretsProvider is used to initialize a Stack with alternative encryption.
 	// Examples:
-	//   - AWS:   "awskms://arn:aws:kms:us-east-1:111122223333:key/1234abcd-12ab-34bc-56ef-1234567890ab?region=us-east-1"
+	//   - AWS:   "awskms:///arn:aws:kms:us-east-1:111122223333:key/1234abcd-12ab-34bc-56ef-1234567890ab?region=us-east-1"
 	//   - Azure: "azurekeyvault://acmecorpvault.vault.azure.net/keys/mykeyname"
 	//   - GCP:   "gcpkms://projects/MYPROJECT/locations/MYLOCATION/keyRings/MYKEYRING/cryptoKeys/MYKEY"
```

Comment:
Are the URI formats for Azure and GCP correct?

Reply:
Yes - at least based on what I see here: https://www.pulumi.com/docs/intro/concepts/secrets/#initializing-a-stack-with-alternative-encryption